Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -99,6 +99,25 @@ public Thickness Padding
                 typeof(WrapPanel),
                 new PropertyMetadata(default(Thickness), LayoutPropertyChanged));
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the last child should fill the remaining space, or not.
+        /// </summary>
+        public bool StretchLastChild
```

Comment:
Did we talk about making this a `StretchChild` enum with like `None`, `First`, and `Last` for values or was that for something else?

Reply:
We don't need to implement `First` for now if we plan to, but would be good to setup the property for that in the future now.

Reply:
We did, but first logic turns out to be slightly more complicated than last logic, and I have no requirement for first logic. 

I'll add the enum.

Reply:
Done 👍